schema/profile.js

```diff
@@ -261,6 +261,9 @@ class Profile extends BaseModel {
         join: {
           from: 'profiles.id',
           to: 'pils.profileId'
+        },
+        filter: f => {
+          return f.skipUndefined().where('establishmentId', f.context().establishmentId);
```

Comment:
What is this for?

Almost certainly nothing wrong with it, I just don't understand it and would appreciate an explanation.